meilisearch-http/tests/auth/payload.rs

```diff
@@ -275,7 +275,7 @@ async fn error_api_key_empty_payload() {
 
 #[actix_rt::test]
 async fn error_api_key_malformed_payload() {
-    let content = r#"{"malormed": "payload""#;
```

Comment:
I don't understand how this test was passing...

Reply:
NP. Thought that was potentially more a bug than a type, but didn't chase down where `content` is re-used...

Will look into what's going on with the failed test when I get cycles (hopefully tonight) and rebase.

Reply:
@Kerollmops I just rebased and removed this commit(it was isolated already). The rest are small docs changes, didn't anticipate a test breaking. 

Will put the `malformed` and test on the list for another day and PR :)